doc/source/reference.rst

```diff
@@ -107,14 +107,17 @@ charts.
 
 However, the OpenStack-Helm project has a repository, `openstack-helm-images`,
 containing a reference implementation for the images. That repository
-holds the images used for the OpenStack-Helm project charts.
-
-These images are built with Docker.
-`openstack-helm-images` provides Dockerfiles for all the non-OpenStack images.
-For the OpenStack images, `openstack-helm-images` only has a shell script, a
-thin wrapper around `LOCI`. `LOCI` is the official OpenStack project to
-build OCI compliant images of OpenStack. It follows a source based deployment
-model.
+holds the images used for the OpenStack-Helm project charts. All its images
+are built with Docker.
+
+`openstack-helm-images` provides Dockerfiles directly for all the
+non-OpenStack images.
+
+For the OpenStack images, situated in `openstack/loci/`,
+`openstack-helm-images` only has a shell script, a thin wrapper around `LOCI`.
```

Comment:
_only is_ ?

Reply:
Maybe I should refactor this whole sentence. Will do tomorrow.